packages/cypress/tests/cypress/e2e/custom-downloads-directory.cy.ts

```diff
@@ -0,0 +1,11 @@
+it(
+  'downloads archives to the user-specified folder',
+  { env: { disableAutoSnapshot: true } },
+  () => {
+    cy.visit('/asset-paths/query-params');
+    const chromaticArchivesDir = `${Cypress.config('downloadsFolder')}/chromatic-archives`;
+    cy.task('directoryExists', chromaticArchivesDir).then((dirExists) => {
```

Comment:
@skitterm thanks for laying the groundwork for this Cypress task, the test was pretty easy to write with that!